app/scripts/controllers/dashboard.ts

```diff
@@ -1,79 +1,64 @@
-            "Carrot cake powder dragée jelly beans. Sugar plum lollipop cotton candy chupa chups liquorice. Donut tootsie roll bonbon halvah icing chupa"},
-          {id: 5, title:"Chips",
-            summary: "Ice cream liquorice biscuit candy canes candy biscuit caramels toffee. Wafer toffee gummies gummi bears muffin lollipop cake croissant gingerbread. Macaroon carrot cake croissant wafer sweet pie sweet roll biscuit chocolate bar. Caramels soufflé bonbon carrot cake. Halvah tiramisu wafer. " +
-            "Carrot cake powder dragée jelly beans. Sugar plum lollipop cotton candy chupa chups liquorice. Donut tootsie roll bonbon halvah icing chupa"
-          },
-          {id: 6, title:"Chocolate",
-            summary: "Ice cream liquorice biscuit candy canes candy biscuit caramels toffee. Wafer toffee gummies gummi bears muffin lollipop cake croissant gingerbread. Macaroon carrot cake croissant wafer sweet pie sweet roll biscuit chocolate bar. Caramels soufflé bonbon carrot cake. Halvah tiramisu wafer. " +
-            "Carrot cake powder dragée jelly beans. Sugar plum lollipop cotton candy chupa chups liquorice. Donut tootsie roll bonbon halvah icing chupa"
-
-          },
-        ];
-
-
-        $scope.hideSuccessMessage = ()  =>  {
-          $scope.message.success = false;//hide message if user wants to write/submit more feedback
-          console.log("from close function " + $scope.message.success);
-        }
+module labsFrontendApp {
+    export interface IDashboardScope {
+        releases: any;
+        feature: any;
+        message: any;
+        feature_sections: any;
+        hideSuccessMessage: any;
+        pickedFeature: any;
+
+    }
+
+    export class DashboardCtrl {
+        // @ngInject
+        constructor(private $scope:IDashboardScope, private releases:GetAllPublicReleases, private toggles:GetTogglesForRelease) {
+
+            var releasePromise = releases.execute();
+            this.getToggles( releasePromise );
+
+            releasePromise.then((releases) => {
+                $scope.releases = releases;
+            });
+
+
+            $scope.feature = {
+                chosen: undefined
+            };
 
-        $scope.pickedFeature = ( pickedFeature ) => {
-          $scope.feature.chosen = pickedFeature;
-        };
+            $scope.message = {
+                success: undefined
+            };
 
+            $scope.hideSuccessMessage = ()  => {
+                $scope.message.success = false;//hide message if user wants to write/submit more feedback
+                console.log("from close function " + $scope.message.success);
+            };
+
+            $scope.pickedFeature = (pickedFeature) => {
+                $scope.feature.chosen = pickedFeature;
+            };
+        }
+
+        /**
+         * @param releases
+         */
+        public getToggles( releases: ng.IPromise<Array<Release>> )
```

Comment:
so this will get the toggles once the releases are available to get toggles for
unfortunately we don't get a release ID yet so I hard coded it to 1 for now